src/cljs/athens/keybindings.cljs

```diff
@@ -78,12 +79,16 @@
                                          (.. e preventDefault)
                                          (if (= index 0)
                                            (swap! state assoc :search/index (dec (count results)))
-                                           (swap! state update :search/index dec)))
+                                           (swap! state update :search/index dec))
+                                         (scrollIfNeeded (.getElementById js/document (str "result-" (:search/index @state)))
```

Comment:
Will probably want to define these HTML ids soon. They are becoming a little unwieldy